tests/goth_tests/conftest.py

```diff
@@ -0,0 +1,93 @@
+"""Pytest configuration file containing the utilities for Dapp Runner goth tests."""
+from datetime import datetime
+from pathlib import Path
+from typing import AsyncGenerator, List, cast
+
+import pytest
+
+from goth.configuration import Override, load_yaml
+from goth.runner import Runner
+from goth.runner.log import configure_logging
+from goth.runner.probe import RequestorProbe
+
+
+def _project_dir() -> Path:
+    package_dir = Path(__file__).parent.parent
+    return package_dir.parent.resolve()
+
+
+def pytest_addoption(parser):
+    """Add optional parameters to pytest CLI invocations."""
+
+    parser.addoption(
+        "--config-path",
+        help="Path to the `goth-config.yml` file. (default: %(default)s)",
+        default=_project_dir() / "tests" / "goth_tests" / "assets" / "goth-config.yml",
+    )
+
+    parser.addoption(
+        "--config-override",
+        action="append",
+        help="Set an override for a value specified in goth-config.yml file. \
+                This argument may be used multiple times. \
+                Values must follow the convention: {yaml_path}={value}, e.g.: \
+                `docker-compose.build-environment.release-tag=0.6.`",
+    )
+
+    parser.addoption(
```

Comment:
uh, I don't think this is needed as it pertains specifically to one of the tests in yapapi